modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_rule_validator.js

```diff
@@ -24,7 +24,11 @@ AUI.add(
 					_isValidAction: function(action) {
 						var instance = this;
 
-						if (instance._isPropertyAction(action.action) && action.target) {
+						if (action.action !== '' && !instance._isPropertyAction(action.action)) {
+							return true;
+						}
+
+						if (action.target) {
```

Comment:
These two `if` returning `true` make the code looks wired. Will be better if you add a `if` statement where you checks if the action is a Property Action and inside the if scope check if the action is valid (`return action.target`) and the else statement always returns true.